### DISCUSSION TITLE
Searching within torrents

### DISCUSSION THREAD

**Discussion by csm10495** [pymedusa/Medusa#9171] [Ideas & Feature requests]
It would be cool if i set Show X S02E01 to wanted and a specific torrent for that episode couldn't be found, for it to (somehow?) create a torrent for a piece of a larger torrent.

So maybe this episode is in the torrent for all of S02 for Show X and as a result it creates a torrent that just grabs that episode from the larger torrent. 

If this granularity isn't possible, maybe just a search mode for a full season/series where these larger torrents could be used to get all episodes instead of each individual episode torrent.

**Comment by p0psicles** [comment]
Config - providers. Set it to search season packs, fall back to episode. 

Or in show page next to season title, click the season manual search button

**Reply by csm10495** [reply]
Will it only download the desired episode then?

**Reply by Rouzax** [reply]
No, but most Private Trackers are not happy if you incomplete download a torrent
